src/models/challenges.d.ts

```diff
@@ -22,6 +22,11 @@ declare namespace socket_api {
          *
          * This is a work in progress. Trust these values at your own risk.
          * */
+
+        export interface RengoGameFlags {
```

Comment:
Is this still `RengoGameFlags` used?  If it is necessary, can you move the `seekgraph` comment below it?  Otherwise intellisense won't know that it's describing `Challenge`

<img width="586" alt="Screen Shot 2022-01-28 at 9 41 21 PM" src="https://user-images.githubusercontent.com/25233703/151649153-669706e7-a313-415d-8645-5d6cefec31af.png">

Reply:
No, RengoGameFlags turned out to be a bad idea, flags on a game need to be individual, not grouped in that way, for ease of translation into DB schema